redbot/cogs/mod/settings.py

```diff
@@ -199,3 +204,19 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context, enabled: bool = None):
+        """Toggle whether to send a message to a user when they are kicked/banned.
```

Comment:
add a new line after `Toggle whether to send a message to a user when they are kicked/banned.` 
So that the second part of this message is properly formatted